electronics/ohms_law.py

```diff
@@ -0,0 +1,32 @@
+# Importing doctest for testing our function
+from doctest import testmod
+
+# Defining function
+
+
+def ohms_law(voltage: float = 0, current: float = 0, resistance: float = 0) -> float:
+
+    """
+    This function calculates the any one
+    of the three ohms_law values voltage,
+    current, resistance, of electronics.
+    >>> ohms_law(voltage=10, resistance=5,current=0 )
+    2.0
+    >>> ohms_law(current=1, resistance=10, voltage=0)
+    10.0
+    """
```

Comment:
The very first thing that this function should do is raise ValueError if 0.0 does not appear once and only once in the three input parameters.  Returning 0 on bad data is not cool.

Reply:
Yes i am thinking about it before even you tell me, so now i will do something for it